subprojects/smoke-test/build.gradle.kts

```diff
@@ -122,11 +119,35 @@ tasks {
         delete(remoteProjects.map { it.outputDirectory })
     }
 
-    withType<SmokeTest>().configureEach {
+    val santaSmokeTests = withType<SmokeTest>().matching { !it.name.startsWith("gradleBuild") }
+    val gradleBuildSmokeTests = withType<SmokeTest>().matching { it.name.startsWith("gradleBuild") }
+    val gradleBuildTestPattern = "**/GradleBuild*SmokeTest*"
```

Comment:
Can you add a comment to `GradleBuildConfigurationCacheSmokeTest` that this class in included in a different task via the name? Or maybe we could use JUnit categories instead. WDYT?

Reply:
Using a junit category sounds good :+1: